frontend/PreferencesPanel.js

```diff
@@ -90,7 +90,7 @@ class PreferencesPanel extends React.Component {
               ref={this._setSelectRef}
               value={themeName}
             >
-              {!showHiddenThemes && (<option value="">{browserName}</option>)}
+              {browserName && (<option value="">{browserName}</option>)}
```

Comment:
Thanks for providing additional context and a demo gif.

I'm still not sure I understand why someone would set a theme in the way shown in your gif above vs using the built-in theme setting. It seems...unnecessarily indirect. Which is fine, except that it adds complexity to code that's already a little more complex than I'd prefer.

I find myself questioning whether the added complexity is justified.